src/Components/Todos.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import NewTodo from "./NewTodo";
+import Todo from "./Todo";
+import { useSelector } from "react-redux";
+
+export default function Todos() {
+  const todos = useSelector((state) => state.todos);
+
+  const [showNewTodoForm, setShowNewTodoForm] = useState(false);
+
+  const handleCreate = () => {
+    setShowNewTodoForm(true);
+  };
+
+  const handleNewSave = () => {
+    setShowNewTodoForm(false);
+  };
+
+  return (
+    <div>
+      <h1>Add Tasks</h1>
+      <button onClick={handleCreate}>Create</button>
+      <br />
+      <br />
+      <div className="todo-container">
+        <div className="todos-row">
+          {showNewTodoForm && (
+            <NewTodo className="todo-item" handleNewSave={handleNewSave} />
+          )}
+          {todos.map((todo, index) => (
+            <Todo
+              className="todo-item"
+              key={index}
```

Comment:
ditto

Reply:
Please have a look at below reference. [Why you should not use index as key](https://react.dev/learn/rendering-lists#why-does-react-need-keys)

Reply:
used todo.id as key